### PR TITLE
Sort bids and show wrap link

### PR DESCRIPTION
**Pull request by DemolaJames**
**Details**
- Sorting of bids by limit price 
- Adding Utilities section to help with wrap/unwrapping or weth, getting Osqth

### CODE REVIEW

File: container/CrabV2/CrabOTC/CrabOTC.tsx [hunk 1]
```diff
@@ -345,24 +346,24 @@ const Withdraw: React.FC = () => {
...
                     </TableCell>
                     <TableCell component="th" scope="row">
-                      {Date.now() < userOtc?.data.bids[bidId].order.expiry && Date.now() < userOtc?.data.expiry ? (
+                      {Date.now() < userOtc?.data.sortedBids[Number(bidId)].order.expiry && Date.now() < userOtc?.data.expiry ? (
                         <BoxLoadingButton
                           sx={{ width: 100 }}
                           size="small"
-                          onClick={() => withdrawCrab(userOtc?.data.bids[bidId], userOtc, i + 1)}
-                          loading={withdrawLoading === i + 1}
+                          onClick={() => withdrawCrab(userOtc?.data.sortedBids[Number(bidId)], userOtc, Number(bidId) + 1)}
```
Comment (sanandnarayan) [inline container/CrabV2/CrabOTC/CrabOTC.tsx:365]: @DemolaJames have you tested that this is working in the inteneded way. Is it picking the right bid amount?
Can you test this on ropsten and post a loom on this PR?

**Review by sanandnarayan — COMMENTED** [review]
Hi Demola, please leave a test video. Thank you

File: utils/crabotc.ts [hunk 2]
```diff
@@ -92,3 +92,14 @@ export const validateBalance = (
 
   return { isValidOrder, response }
 }
+
+export const sortBidsForBidArray = (bids: Array<CrabOTCBid>, isSelling: boolean) => {
+  const sortedBids = bids.sort((a, b) => {
+    if (b.order.price === a.order.price) return Number(a.order.nonce) - Number(b.order.nonce)
```
Comment (sanandnarayan) [inline utils/crabotc.ts:98]: So when they are equal, you are giving preference to the order that came in first?

File: container/CrabV2/CrabOTC/CrabOTC.tsx (hunk 1 above)
Comment (DemolaJames) [reply container/CrabV2/CrabOTC/CrabOTC.tsx:365]: Yes i tested and it picks the right bid amount...will do a loom

File: utils/crabotc.ts (hunk 2 above)
Comment (DemolaJames) [reply utils/crabotc.ts:98]: Yes, will so this in the loom

**Pull request merged**